.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
+rvm:
+  - 2.4.1
```

Comment:
Will this still work with 2..2? That would keep this gem current with the floor version of `solidus` itself

Reply:
[Travis in Solidus](https://github.com/solidusio/solidus/blob/master/.travis.yml ) is using 2.3.0. We should probably stay consistent with our testing.

Reply:
It doesn’t matter that much actually. I’m fine with either 2.4 or 2.3